geniza/corpus/tests/test_sitemaps.py

```diff
@@ -0,0 +1,40 @@
+from geniza.corpus.models import Document
+from geniza.corpus.sitemaps import DocumentScholarshipSitemap, DocumentSitemap
+
+
+class TestDocumentSitemap:
+    def test_items(self, document):
+        # Ensure that documents are supressed if they aren't public
+        assert document.status == Document.PUBLIC
+        sitemap = DocumentSitemap()
+        assert len(sitemap.items()) == 1
```

Comment:
would prefer a more robust test (object / pk in list or not in list)

Reply:
f37959c90b06931a1f954140fc1fbf2c34891ebd ✅